skcosmo/preprocessing/flexible_scaler.py

```diff
@@ -113,19 +124,28 @@ def __init__(self):
         """Initialize KernelFlexibleCenterer."""
         pass
 
-    def fit(self, K=None, y=None, K_fit_rows=None, K_fit_all=None):
+    def fit(self, K=None, y=None, sample_weight=None, K_fit_rows=None, K_fit_all=None):
```

Comment:
Does this fit with the sklearn-style syntax for KernelCenterer?

Reply:
sklearn's `KernelCenterer` doesn't allow for sample weights, but the `sample_weight` argument is consistent with models that do allow it in the `fit` call, e.g., `Ridge`. The `fit` call for sklearn's `KernelCenterer` also doesn't take `K_fit_rows` or `K_fit_all`, but those were there already.